.github/workflows/tools.yml

```diff
@@ -0,0 +1,60 @@
+name: Build Tools
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Tagged commit to build tools against'
+        required: true
+        type: string
+      promote:
+        description: 'Should this build be promoted to the official boot-tools?'
+        required: false
+        type: boolean
+
+jobs:
+  build-publish:
+    name: Build boot tools
+    runs-on: ubuntu-latest
+    steps:
+    - id: auth
+      uses: google-github-actions/auth@v1
+      with:
+        credentials_json: ${{ secrets.GCR_SERVICE_KEY }} # TODO: we need a new key to allow uploads
```

Comment:
@sjonpaulbrown @gomisha could one of your help create and add a new secret for this usecase?